overlay/files/system/oem/08_grub.yaml

```diff
@@ -102,7 +102,7 @@ stages:
       name: "Grub branding"
       if: '[ -e "/etc/kairos/branding/grubmenu.cfg" ]'
       commands:
-      - cp -rfv /etc/kairos/branding/grubmenu.cfg /tmp/mnt/STATE/grubmenu
+        - cp -rfv /etc/kairos/branding/grubmenu.cfg /tmp/mnt/STATE/grubmenu
```

Comment:
this doesn't look right :thinking:

Reply:
@mudler as in, it looks different from https://github.com/kairos-io/kairos/pull/1426/files or as in, maybe that was not the actual fix?

Reply:
the diff shows only a whitespace, only on the branch.. should look a bit different?

Reply:
LOL I see what you mean ... it's already there ... let me check if I based it from teh right one :/